scipy/optimize/optimize.py

```diff
@@ -276,6 +276,7 @@ def rosen(x):
     ----------
     x : array_like
         1-D array of points at which the Rosenbrock function is to be computed.
+        n-dimensional arrays broadcast using the first component as variables.
```

Comment:
I don't understand what "first component as variables" means.

Reply:
How about:
```n-dimensional arrays broadcast using the first dimension as function variables.```
?

Reply:
ISTM this can be dropped altogether, the example that follows is plenty enough.

Reply:
ok!